apps/api/src/scraper/scrapeURL/lib/urlSpecificParams.ts

```diff
@@ -0,0 +1,78 @@
+import { InternalOptions } from "..";
+import { ScrapeOptions } from "../../../controllers/v1/types";
+
+export type UrlSpecificParams = {
+    scrapeOptions: Partial<ScrapeOptions>,
+    internalOptions: Partial<InternalOptions>,
+};
+
+const docsParam: UrlSpecificParams = {
```

Comment:
why are we overriding the headers?

Reply:
I don't know, took this from WebScraper. @nickscamara any idea?

Reply:
It is old. No need for it.

Reply:
YEETED (YOTE?)